components/navbar.vue

```diff
@@ -1,5 +1,17 @@
 <template>
-  <nav>
+  <section>
+    <nav class="fixed z-100 w-full backdrop-blur-sm backdrop-filter py-4 hidden md:flex justify-between py-4 px-4 items-center">
+      <aside class="font-bold text-3xl tracking-widest hover:(scale-125) transform px-8 py-2">
+        <nuxt-link to="/">SDG</nuxt-link>
+      </aside>
+      <ul class="ml-auto flex-row md:flex list-none">
+        <li v-for="page in pages" :key="page.url" class="hover:(font-bold scale-125) transform px-8">
```

Comment:
A transition of 0.5s could be added -- font size changes suddenly in current version.

Reply:
Sounds good